biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -16,9 +17,19 @@ export const SimulatePage = () => {
     setSimulationResults(results);
   }
 
+  const history = useHistory();
+
+  const redirect = () => {  
+    history.goBack();
```

Comment:
Can you change the user URL to the home page instead of relying on history to go back?